skbase/base/_base.py

```diff
@@ -216,6 +217,42 @@ def get_param_defaults(cls):
         }
         return default_dict
 
+    def get_params(self, deep=True):
+        """Get a dict of parameters values for this object.
+
+        Parameters
+        ----------
+        deep : bool, default=True
+            If True, will return a dict of parameter name : value for this object,
+            including parameters of components (= BaseObject-valued parameters).
+            If False, will return a dict of parameter name : value for this object,
+            but not include parameters components.
```

Comment:
I think it should be "but not include paremeters of components" or  "but not include component parameters".

Reply:
ah, of course. Or `parameters' components` which is confusing, so I changed it to your first suggestion.